test/manual/kerberos.test.js

```diff
@@ -0,0 +1,352 @@
+'use strict';
+
+const format = require('util').format;
+const test = require('../functional/shared').assert;
+const { MongoClient } = require('../../src');
+
+// You need to set up the kinit tab first
+// https://wiki.mongodb.com/pages/viewpage.action?title=Testing+Kerberos&spaceKey=DH
+// kinit -p drivers@LDAPTEST.10GEN.CC
+// password: (not shown)
+
+describe('Kerberos', function () {
+  if (process.env.MONGODB_URI == null) {
+    console.log('skipping Kerberos tests, MONGODB_URI environment variable is not defined');
+    return;
+  }
+  if (process.platform === 'win32') {
+    console.log('Only running Win32 tests');
```

Comment:
This message looks wrong, also we would eventually want to run this against windows

Reply:
This is meant to be the equivalent of `{ metadata: { os: '!win32' } }` without depending on our custom runner. There's a set of tests in the second `describe` that are win32-specific.